ci/toolchain_manager.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Builds and publishes kubesaw components to a quay repository.  Used as a part
+# of the CI process for testing in openshift-ci.
+
+set -e
+
+# parse input
+BRANCH=${BRANCH:-pubviewer-mvp}
+QUAY_NAMESPACE=${QUAY_NAMESPACE:-konflux-workspaces}
+TAG=${2:-e2etest}
+KUBECLI=${KUBECLI:-kubectl}
+
+HOST_OPERATOR_REPO=${HOST_OPERATOR_REPO:-https://github.com/filariow/host-operator.git}
+MEMBER_OPERATOR_REPO=${MEMBER_OPERATOR_REPO:-https://github.com/codeready-toolchain/member-operator.git}
+TOOLCHAIN_E2E_REPO=${TOOLCHAIN_E2E_REPO:-https://github.com/sadlerap/toolchain-e2e.git}
+REGISTRATION_SERVICE_REPO=${REGISTRATION_SERVICE_REPO:-https://github.com/filariow/registration-service}
```

Comment:
I would fork these into our workspace github project, and give ourself the ability to change the branches for each to be able to run test on work in progress. This would avoid having to use personal repos like above

Reply:
Agreed, though I think I'll punt that work to a future PR - I'd like to get something working first, since that unblocks openshift-ci from working.

Reply:
Story?

Reply:
https://issues.redhat.com/browse/ASC-533